testing/regress/hpcc/util/ecl/command.py

```diff
@@ -78,7 +78,9 @@ def runCmd(self, cmd, cluster, eclfile, report, **kwargs):
 
             args = args + eclfile.getStoredInputParameters()
 
-            args.append('-XTargetIP=' + server)
+            if '-XTargetIP' not in "".join(args):
+                logging.debug("%3d. results:'%s'", eclfile.getTaskId(),  "'-XTargetIP' is not in args")
+                args.append('-XTargetIP=' + server)
```

Comment:
I'm not clear what this is doing, I expected any mention of TargetIP to be removed.
i.e. comment from JIRA:
>Since "TargetIP" becomes a normal parameter, you can override on command line or in config file like any other.